### ISSUE TITLE
Pinpoint faulty mapping

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo#4120]
At least one of the following mappings is false:

```
EquivalentClasses(<http://linkedlifedata.com/resource/umls/id/C1862937> <http://purl.obolibrary.org/obo/MONDO_0007105>)
EquivalentClasses(<http://purl.obolibrary.org/obo/DOID_0060213> <http://purl.obolibrary.org/obo/MONDO_0007105>)
EquivalentClasses(<http://purl.obolibrary.org/obo/MONDO_0007105> <http://purl.obolibrary.org/obo/OMIM_105550>)

EquivalentClasses(<http://linkedlifedata.com/resource/umls/id/C1862937> <http://purl.obolibrary.org/obo/MONDO_0010857>)
EquivalentClasses(<http://purl.obolibrary.org/obo/MONDO_0010857> <http://purl.obolibrary.org/obo/OMIM_600274>)
EquivalentClasses(<http://purl.obolibrary.org/obo/MONDO_0010857> <http://www.orpha.net/ORDO/Orphanet_100069>)
EquivalentClasses(<http://purl.obolibrary.org/obo/GARD_0010792> <http://purl.obolibrary.org/obo/MONDO_0010857>)
```

And this looks suspicious:

```
EquivalentClasses(<http://linkedlifedata.com/resource/umls/id/C1862937> <http://purl.obolibrary.org/obo/MONDO_0007105>)
EquivalentClasses(<http://linkedlifedata.com/resource/umls/id/C1862937> <http://purl.obolibrary.org/obo/MONDO_0010857>)
```

**Comment by nicolevasilevsky** [comment]
UMLS:C1862937 was retired as of the 2018AA version of the UMLS. It was merged into UMLS:C3888102.

I made UMLS:C1862937 MONDO:equivalentObsolete in Mondo.

Maybe this is the issue?

**Comment by matentzn** [comment]
yes it would be! Thank you!
